mlir/lib/Dialect/LLVMIR/IR/LLVMAttrs.cpp

```diff
@@ -37,14 +37,28 @@ void LLVMDialect::registerAttributes() {
       >();
 }
 
+//===----------------------------------------------------------------------===//
+// MDNodeAttr
+//===----------------------------------------------------------------------===//
+
+bool MDNodeAttr::classof(Attribute attr) {
+  return llvm::isa<DIBasicTypeAttr, DICompileUnitAttr, DICompositeTypeAttr,
```

Comment:
Can this list be simplified by using DINodeAttr? That way there's no need to add to two places when adding a DINodeAttr?

Reply:
You're right. I'll fix this.